client/dashboard/domains/domain-transfer/transfer-domain-to-other-user.tsx

```diff
@@ -32,7 +36,14 @@ export type TransferFormData = {
 	user: string;
 };
 
-const createFields = ( users: SiteUser[] ): Field< TransferFormData >[] => [
+const getWpcomUserId = ( user: WpcomSiteUser ) => user.linked_user_ID ?? user.ID;
+
+const getUserDisplayName = ( user: WpcomSiteUser ) => {
+	const { first_name, last_name, nice_name } = user;
+	return first_name && last_name ? `${ first_name } ${ last_name } (${ nice_name })` : nice_name;
```

Comment:
I don't think we can [simply concatenate first and last names like this](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/). We have the "public display name" field for that sort of thing.

<img width="1172" height="1020" alt="CleanShot 2025-12-15 at 14 00 19@2x" src="https://github.com/user-attachments/assets/87f5d2f0-2fc1-41d8-8f89-3b066235fcb4" />

I'm not exactly sure what "nice name" is. But I think if we want to show the username to help disambiguate then it should something more like

```suggestion
	const { name, login } = user;
	return name ? `${ name } (${ login })` : login;
```